services/server/src/apiv2.yaml

```diff
@@ -905,13 +905,6 @@ components:
                     chainId: '11155111'
                     address: '0x2738d13E81e30bC615766A0410e7cF199FD59A83'
                     verifiedAt: '2024-07-24T12:00:00Z'
-                pagination:
-                  currentPage: 3
-                  totalPages: 243
-                  resultsPerPage: 5000
-                  totalResults: 24245
-                  hasNextPage: true
-                  hasPreviousPage: true
```

Comment:
Why this was removed?

Reply:
We don't have it anymore, because we changed the pagination style. This is only here because I copied the api v2 spec from Stoplight to be up-to-date. Seems like this change was forgotten earlier.